scripts/components/timer.js

```diff
@@ -153,8 +178,9 @@ var Selector = React.createClass({
 
         return <div className="gotr-selector">
             <div className="gotr-selector-box">
-                <input
+                <input type='number'
                     onChange={this.updateMins}
+                    onKeyDown={this.startTiming}
```

Comment:
This is where you'd add `ref="minsInput"`. `this.startTiming` doesn't appear to exist. I prefer to use double-quotes in markup.